snuba/utils/health_info.py

```diff
@@ -107,7 +107,8 @@ def get_health_info(thorough: Union[bool, str]) -> HealthInfo:
     payload = json.dumps(body)
     if status != 200:
         metrics.increment("healthcheck_failed", tags=metric_tags)
-        logger.error(f"Snuba health check failed! Tags: {metric_tags}")
+        if not down_file_exists:
+            logger.error(f"Snuba health check failed! Tags: {metric_tags}")
```

Comment:
I would just info log it if the down file exists, it's useful log info, just not an error